changelog.py

```diff
@@ -0,0 +1,438 @@
+                iPhone_list.append(iPhone_line)
+            if "webcompat" in label_names\
+                    or "audio" in label_names\
+                    or "feature/video" in label_names\
+                    and "tests" not in label_names:
+                webcompat_line = " - " +\
+                    issue_title + ".(#" + str(issue.number) + ")"
+                webcompat_list.append(webcompat_line)
+            if "feature/sync" in label_names:
+                sync_line = " - " +\
+                    issue_title + ".(#" + str(issue.number) + ")"
+                sync_list.append(sync_line)
+
+    if iPad_list:
+        print("## iPad Specific Fixes")
+        for line in sorted(iPad_list):
+            print(line)
+        print("")
+
+    if iPhone_list:
+        print("## iPhone Specific Fixes")
+        for line in sorted(iPhone_list):
+            print(line)
+        print("")
+
+    if crash_list:
+        print("## Crash Fixes")
+        for line in sorted(crash_list):
+            print(line)
+        print("")
+
+    if shields_list:
+        print("## Shields Fixes")
+        for line in sorted(shields_list):
+            print(line)
+        print("")
+
+    if sync_list:
+        print("## Sync")
+        for line in sorted(sync_list):
+            print(line)
+        print("")
+
+    if regression_list:
+        print("## Regression fixes")
+        for line in sorted(regression_list):
+            print(line)
+        print("")
+
+    if webcompat_list:
+        print("## Webcompat fixes")
+        for line in sorted(webcompat_list):
+            print(line)
+        print("")
+
+    if general_list:
+        print("## Other Fixes")
+        for line in sorted(general_list):
+            print(line)
+        print("")
+
+    return
+
+
+def android_changelog():
+
+    andrky = sorted(android_milestone.keys())[0]
+    for issue in android_repo.get_issues(
+                                         milestone=android_milestone[andrky],
+                                         sort="created", direction="asc",
+                                         state="closed"):
+        if('pull' not in issue.html_url):
+            original_issue_title = issue.title
+            issue_title = original_issue_title
+            if(original_issue_title[0].islower()):
+                lower = original_issue_title[0]
+                upper = original_issue_title[0].upper()
+                issue_title = original_issue_title.replace(lower, upper, 1)
+
+            labels = issue.get_labels()
+            label_names = []
+            for label in labels:
+                label_names.append(label.name)
+            if "crash" in label_names and "tests" not in label_names:
+                crash_line = " - " +\
+                    issue_title + ".(#" + str(issue.number) + ")"
+                crash_list.append(crash_line)
+            if "crash" not in label_names and "webcompat" not in label_names\
```

Comment:
A nice way to clear up trailing slashes is just to have an opening parenthesis, then you can do multiple lines until you close it.

Reply:
without the need of a slash at the end of each line.